src/main/webapp/homepage_script.js

```diff
@@ -43,48 +44,15 @@ function addUIClickHandlers() {
     }
 }
 
-/* Function adds all the necessary tasks 'click' event listeners*/
```

Comment:
This was also not deleted, but moved further down the script file so it's easier to read (it's now closer to the function that calls the function).